src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -74,6 +74,18 @@ public final class SunEC extends Provider {
      */
     private static final boolean useNativeECKeyGen = NativeCrypto.isAlgorithmEnabled("jdk.nativeECKeyGen", "SunEC");
 
+    /* The property 'jdk.nativeXDHKeyAgreement' is used to control enablement of the native
+     * XDH key agreement. XDH key agreement is only supported in OpenSSL 1.1.1 and above.
+     */
+    private static final boolean useNativeXDHKeyAgreement =
+                                        NativeCrypto.isAlgorithmEnabled("jdk.nativeXDHKeyAgreement", "SunEC");
```

Comment:
This line and line 87 have excessive indentation (4 spaces more than the preceding line is sufficient).

Reply:
Fixed.